### PR TITLE
Use Ruff for formatting

### PR DESCRIPTION
**Pull request by mattdangerw**
Keras is doing it, and we should follow suit. https://github.com/keras-team/keras/pull/20445

But also it's simpler and faster!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -216,7 +216,7 @@ so you aren't waiting around forever!
 
 ## Formatting Code
 
-We use `flake8`, `isort` and `black` for code formatting.  You can run
+Keras uses [Ruff](https://docs.astral.sh/ruff/) to format the code. You can run
```
Comment (SamanehSaadat) [inline CONTRIBUTING.md:219]: Don't we want to say `KerasHub` instead of `Keras` here?

File: keras_hub/src/layers/modeling/rms_normalization.py [hunk 2]
```diff
@@ -21,12 +22,13 @@ def __init__(self, input_dim):
...
-        """
-        Applies RMS normalization to the input tensor.
+        """Applies RMS normalization to the input tensor.
+
         Args:
-            x: KerasTensor. Input tensor of shape (batch_size, input_dim).
+            x: Input tensor of shape (batch_size, input_dim).
+
         Returns:
-            KerasTensor: The RMS-normalized tensor of the same shape (batch_size, input_dim),
```
Comment (SamanehSaadat) [inline keras_hub/src/layers/modeling/rms_normalization.py:29]: Did `Ruff` automatically removed `KerasTensor` from here or you removed it manually? 🤔

**Review by SamanehSaadat — APPROVED** [review]
Thanks, Matt!

File: keras_hub/src/layers/modeling/rms_normalization.py (hunk 2 above)
Comment (mattdangerw) [reply keras_hub/src/layers/modeling/rms_normalization.py:29]: I removed it manually while fixing line length issues. We can return either a kerastensor (symbolic tensor) or torch or jax or tf tensor. Better to just say tensor and shorter for line length.

**Pull request merged**